test/models/buildingModels/building_list_test.dart

```diff
@@ -9,10 +9,19 @@ class MockAssetBundle extends Mock implements AssetBundle{}
 void main() {
   TestWidgetsFlutterBinding.ensureInitialized();
   group('BuildingList', () {
-    test('readBuildingFile() creates list of BuildingInformation with a specific length', () async {
+    test('readBuildingFile() creates list of BuildingInformation with the same length as the asset file.', () async {
+      
+      // Build list
       BuildingList buildingList = new BuildingList();
       await buildingList.readBuildingFile();
-      expect(buildingList.getListOfBuildings().length, 10);
+
+      // Get number of lines from source
+      String file = await rootBundle.loadString('assets/campus_buildings_info.txt');
+      String linebreak = '\n';
+      int numLines = linebreak.allMatches(file).length;
+
+      // Compare lengths
+      expect(buildingList.getListOfBuildings().length, numLines);
```

Comment:
friendly reminder that in the future we'll be using afc561d6ac7f6c45a6be4a69260e70de900acced so tests will need to be updated

Reply:
It should be easy to update tests once we use json